pkg/pipeline/ingest/ingest_file.go

```diff
@@ -38,7 +38,7 @@ const delaySeconds = 10
 // Ingest ingests entries from a file and resends the same data every delaySeconds seconds
 func (r *IngestFile) Ingest(process ProcessFunction) {
```

Comment:
why use `r` .... if the is an ingest for files ??? U can use `i` or `if` but what `r` ???